### ISSUE TITLE
UNB xlinux/plinux NoRouteToHostException: No route to host

### ISSUE THREAD

**Issue by pshipton** [eclipse/openj9#7820]
https://ci.eclipse.org/openj9/job/Test_openjdk11_j9_sanity.system_x86-64_linux_xl_Nightly/198/
https://ci.eclipse.org/openj9/job/Test_openjdk8_j9_sanity.system_x86-64_linux_Nightly/203/
```
CL1 stderr java.rmi.ConnectIOException: Exception creating connection to: 192.168.10.185; nested exception is: 
CL1 stderr 	java.net.NoRouteToHostException: No route to host (Host unreachable)
CL1 stderr 	at java.rmi/sun.rmi.transport.tcp.TCPEndpoint.newSocket(TCPEndpoint.java:631)
CL1 stderr 	at java.rmi/sun.rmi.transport.tcp.TCPChannel.createConnection(TCPChannel.java:209)
CL1 stderr 	at java.rmi/sun.rmi.transport.tcp.TCPChannel.newConnection(TCPChannel.java:196)
CL1 stderr 	at java.rmi/sun.rmi.server.UnicastRef.invoke(UnicastRef.java:129)
CL1 stderr 	at java.management.rmi/javax.management.remote.rmi.RMIServerImpl_Stub.newClient(Unknown Source)
CL1 stderr 	at java.management.rmi/javax.management.remote.rmi.RMIConnector.getConnection(RMIConnector.java:2105)
CL1 stderr 	at java.management.rmi/javax.management.remote.rmi.RMIConnector.connect(RMIConnector.java:321)
CL1 stderr 	at java.management/javax.management.remote.JMXConnectorFactory.connect(JMXConnectorFactory.java:270)
CL1 stderr 	at java.management/javax.management.remote.JMXConnectorFactory.connect(JMXConnectorFactory.java:229)
CL1 stderr 	at net.adoptopenjdk.test.jlm.remote.ServerConnector.getServerConnection(ServerConnector.java:115)
CL1 stderr 	at net.adoptopenjdk.test.jlm.remote.ServerConnector.<init>(ServerConnector.java:62)
CL1 stderr 	at net.adoptopenjdk.test.jlm.remote.ClassProfiler.<init>(ClassProfiler.java:62)
CL1 stderr 	at net.adoptopenjdk.test.jlm.remote.ClassProfiler.main(ClassProfiler.java:92)
CL1 stderr Caused by: java.net.NoRouteToHostException: No route to host (Host unreachable)
CL1 stderr 	at java.base/java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:242)
CL1 stderr 	at java.base/java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:224)
CL1 stderr 	at java.base/java.net.SocksSocketImpl.connect(SocksSocketImpl.java:403)
CL1 stderr 	at java.base/java.net.Socket.connect(Socket.java:609)
CL1 stderr 	at java.base/java.net.Socket.connect(Socket.java:558)
CL1 stderr 	at java.base/java.net.Socket.<init>(Socket.java:454)
CL1 stderr 	at java.base/java.net.Socket.<init>(Socket.java:231)
CL1 stderr 	at java.rmi/sun.rmi.transport.tcp.TCPDirectSocketFactory.createSocket(TCPDirectSocketFactory.java:40)
CL1 stderr 	at java.rmi/sun.rmi.transport.tcp.TCPEndpoint.newSocket(TCPEndpoint.java:613)
CL1 stderr 	... 12 more
```

**Comment by pshipton** [comment]
@jdekonin I'll disable ub16x64j99 for now.

**Comment by pshipton** [comment]
Similar problem on cent6-x64-1. Disabling this machine as well
https://ci.eclipse.org/openj9/job/Test_openjdk8_j9_extended.functional_x86-64_linux_Nightly/202
```
===============================================
Running test TestManagementAgent_SE80_0 ...
===============================================
TestManagementAgent_SE80_0 Start Time: Thu Nov 21 02:06:33 2019 Epoch Time (ms): 1574319993255
variation: NoOptions
JVM_OPTIONS: -Xcompressedrefs 
[IncludeExcludeTestAnnotationTransformer] [INFO] exclude file is /home/jenkins/workspace/Test_openjdk8_j9_extended.functional_x86-64_linux_Nightly/openjdk-tests/TestConfig/../../jvmtest/TestConfig/resources/excludes/latest_exclude_8.txt
...
... TestNG 6.14.2 by Cédric Beust (cedric@beust.com)
...

java.rmi.ConnectIOException: Exception creating connection to: 192.168.10.188; nested exception is: 
	java.net.NoRouteToHostException: No route to host (Host unreachable)
	at sun.rmi.transport.tcp.TCPEndpoint.newSocket(TCPEndpoint.java:631)
	at sun.rmi.transport.tcp.TCPChannel.createConnection(TCPChannel.java:216)
	at sun.rmi.transport.tcp.TCPChannel.newConnection(TCPChannel.java:202)
	at sun.rmi.server.UnicastRef.invoke(UnicastRef.java:129)
	at java.rmi.server.RemoteObjectInvocationHandler.invokeRemoteMethod(RemoteObjectInvocationHandler.java:227)
	at java.rmi.server.RemoteObjectInvocationHandler.invoke(RemoteObjectInvocationHandler.java:179)
	at com.sun.proxy.$Proxy5.newClient(Unknown Source)
	at javax.management.remote.rmi.RMIConnector.getConnection(RMIConnector.java:2430)
	at javax.management.remote.rmi.RMIConnector.connect(RMIConnector.java:308)
	at javax.management.remote.JMXConnectorFactory.connect(JMXConnectorFactory.java:270)
	at org.openj9.test.attachAPI.TestManagementAgent.verifyManagementAgent(TestManagementAgent.java:155)
	at org.openj9.test.attachAPI.TestManagementAgent.test_startLocalManagementAgent(TestManagementAgent.java:116)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:648)
	at org.testng.TestRunner.run(TestRunner.java:505)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.TestNG.privateMain(TestNG.java:1354)
	at org.testng.TestNG.main(TestNG.java:1323)
Caused by: java.net.NoRouteToHostException: No route to host (Host unreachable)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:607)
	at java.net.Socket.connect(Socket.java:556)
	at java.net.Socket.<init>(Socket.java:452)
	at java.net.Socket.<init>(Socket.java:229)
	at sun.rmi.transport.proxy.RMIDirectSocketFactory.createSocket(RMIDirectSocketFactory.java:40)
	at sun.rmi.transport.proxy.RMIMasterSocketFactory.createSocket(RMIMasterSocketFactory.java:148)
	at sun.rmi.transport.tcp.TCPEndpoint.newSocket(TCPEndpoint.java:613)
	... 35 more
```

**Comment by pshipton** [comment]
https://ci.eclipse.org/openj9/job/Test_openjdk13_j9_extended.functional_x86-64_linux_Nightly/96/
ub16x64j99
TestManagementAgent_0

**Comment by pshipton** [comment]
https://ci.eclipse.org/openj9/job/Test_openjdk11_j9_sanity.system_x86-64_linux_xl_Nightly/200
https://ci.eclipse.org/openj9/job/Test_openjdk13_j9_sanity.system_x86-64_linux_xl_Nightly/97
ub16x64j97

**Comment by pshipton** [comment]
https://ci.eclipse.org/openj9/job/Test_openjdknext_j9_extended.functional_x86-64_linux_OpenJDK/64
ub16x64j95

**Comment by pshipton** [comment]
The UNB plinux machines are online now, and it's happening on these machines as well.
https://ci.eclipse.org/openj9/job/Test_openjdk13_j9_sanity.system_ppc64le_linux_Nightly/108
ub16p8j91

**Comment by pshipton** [comment]
@jdekonin I've set this high priority as it occurs in every nightly build. I didn't record all the recent xlinux failures that occurred over the weekend, but they occurred on the same machines recorded in previous comments.

**Comment by pshipton** [comment]
https://ci.eclipse.org/openj9/job/Test_openjdk13_j9_extended.functional_ppc64le_linux_Nightly/108/
ub16p8j91

https://ci.eclipse.org/openj9/job/Test_openjdknext_j9_extended.functional_ppc64le_linux_OpenJDK/76/
ub16p8j92

**Comment by pshipton** [comment]
https://ci.eclipse.org/openj9/job/Test_openjdk13_j9_extended.functional_x86-64_linux_xl_Nightly/99/
ub16x64j97
This one just hung and was killed.
``
03:59:20  [JvmCpuMonitorMXBeanTest] [INFO]  Starting JvmCpuMonitorMXBean API tests....
03:59:20  [JvmCpuMonitorMXBeanTest] [INFO]  ------------------------------------------
03:59:20  [JvmCpuMonitorMXBeanTest] [INFO] Starting Remote Server to Monitor!
03:59:20  [JvmCpuMonitorMXBeanTest] [INFO] /home/jenkins/workspace/Test_openjdk13_j9_extended.functional_x86-64_linux_xl_Nightly/openjdkbinary/j2sdk-image/bin/java -classpath /home/jenkins/workspace/Test_openjdk13_j9_extended.functional_x86-64_linux_xl_Nightly/openjdk-tests/TestConfig/../../jvmtest/TestConfig/resources:/home/jenkins/workspace/Test_openjdk13_j9_extended.functional_x86-64_linux_xl_Nightly/openjdk-tests/TestConfig/../TestConfig/lib/testng.jar:/home/jenkins/workspace/Test_openjdk13_j9_extended.functional_x86-64_linux_xl_Nightly/openjdk-tests/TestConfig/../TestConfig/lib/jcommander.jar:/home/jenkins/workspace/Test_openjdk13_j9_extended.functional_x86-64_linux_xl_Nightly/openjdk-tests/TestConfig/../../jvmtest/functional/JLM_Tests/jlm_tests.jar -Xnocompressedrefs -Xjit -Xgcpolicy:optthruput -Xnocompressedrefs  -Dcom.sun.management.jmxremote.port=9999 -Dcom.sun.management.jmxremote.authenticate=false -Dcom.sun.management.jmxremote.ssl=false -Djava.lock.file=/tmp/tmp6808195391270670905.lock org.openj9.test.management.RemoteMonitor 
Cancelling nested steps due to timeout
09:31:36  Sending interrupt signal to process
```

**Comment by pshipton** [comment]
https://ci.eclipse.org/openj9/job/Test_openjdk13_j9_sanity.system_ppc64le_linux_Nightly/110/
ub16p8j95
https://ci.eclipse.org/openj9/job/Test_openjdk11_j9_sanity.system_x86-64_linux_Nightly/200/
ub16x64j92
https://ci.eclipse.org/openj9/job/Test_openjdk13_j9_extended.functional_ppc64le_linux_Nightly/109/
ub16p8j91
https://ci.eclipse.org/openj9/job/Test_openjdknext_j9_extended.functional_ppc64le_linux_OpenJDK/77/
ub16p8j96
https://ci.eclipse.org/openj9/job/Test_openjdk11_j9_extended.functional_x86-64_linux_Nightly/197/
ub16x64j95

**Comment by pshipton** [comment]
https://ci.eclipse.org/openj9/job/Test_openjdk11_j9_sanity.system_ppc64le_linux_Nightly/213/
ub16p8j91
https://ci.eclipse.org/openj9/job/Test_openjdk11_j9_sanity.system_x86-64_linux_Nightly/201/
ub16x64j95
https://ci.eclipse.org/openj9/job/Test_openjdk11_j9_extended.functional_ppc64le_linux_Nightly/214/
ub16p8j95
https://ci.eclipse.org/openj9/job/Test_openjdk8_j9_extended.functional_ppc64le_linux_Nightly/212/
ub16p8j94
https://ci.eclipse.org/openj9/job/Test_openjdk11_j9_extended.functional_x86-64_linux_Nightly/198/
ub16x64j92

**Comment by pshipton** [comment]
https://ci.eclipse.org/openj9/job/Test_openjdk13_j9_extended.functional_ppc64le_linux_Nightly/110
ub16p8j91
https://ci.eclipse.org/openj9/job/Test_openjdk11_j9_sanity.system_x86-64_linux_xl_Nightly/205
ub16x64j97
https://ci.eclipse.org/openj9/job/Test_openjdk11_j9_extended.functional_ppc64le_linux_Nightly/215/
ub16p8j94
https://ci.eclipse.org/openj9/job/Test_openjdk8_j9_extended.functional_ppc64le_linux_Nightly/213/
ub16p8j92
https://ci.eclipse.org/openj9/job/Test_openjdknext_j9_extended.functional_ppc64le_linux_OpenJDK/78/
ub16p8j96
https://ci.eclipse.org/openj9/job/Test_openjdk8_j9_extended.functional_x86-64_linux_Nightly/209/
ub16x64j97

@jdekonin

**Comment by pshipton** [comment]
https://ci.eclipse.org/openj9/job/Test_openjdknext_j9_extended.functional_x86-64_linux_xl_OpenJDK/68
ub16x64j95
```
06:28:04  [JvmCpuMonitorMXBeanTest] [INFO]  ------------------------------------------
06:28:04  [JvmCpuMonitorMXBeanTest] [INFO]  Starting JvmCpuMonitorMXBean API tests....
06:28:04  [JvmCpuMonitorMXBeanTest] [INFO]  ------------------------------------------
06:28:04  [JvmCpuMonitorMXBeanTest] [INFO] Starting Remote Server to Monitor!
06:28:04  [JvmCpuMonitorMXBeanTest] [INFO] /home/jenkins/workspace/Test_openjdknext_j9_extended.functional_x86-64_linux_xl_OpenJDK/openjdkbinary/j2sdk-image/bin/java -classpath /home/jenkins/workspace/Test_openjdknext_j9_extended.functional_x86-64_linux_xl_OpenJDK/openjdk-tests/TestConfig/../../jvmtest/TestConfig/resources:/home/jenkins/workspace/Test_openjdknext_j9_extended.functional_x86-64_linux_xl_OpenJDK/openjdk-tests/TestConfig/../TestConfig/lib/testng.jar:/home/jenkins/workspace/Test_openjdknext_j9_extended.functional_x86-64_linux_xl_OpenJDK/openjdk-tests/TestConfig/../TestConfig/lib/jcommander.jar:/home/jenkins/workspace/Test_openjdknext_j9_extended.functional_x86-64_linux_xl_OpenJDK/openjdk-tests/TestConfig/../../jvmtest/functional/JLM_Tests/jlm_tests.jar -Xnocompressedrefs -Xjit -Xgcpolicy:optthruput -Xnocompressedrefs  -Dcom.sun.management.jmxremote.port=9999 -Dcom.sun.management.jmxremote.authenticate=false -Dcom.sun.management.jmxremote.ssl=false -Djava.lock.file=/tmp/tmp8738976736810511091.lock org.openj9.test.management.RemoteMonitor 
Calling Pipeline was cancelled
Calling Pipeline was cancelled
Calling Pipeline was cancelled
Calling Pipeline was cancelled
10:24:34  Sending interrupt signal to process
```

**Comment by jdekonin** [comment]
None of the machine in the last 2 comments have changed ip addresses and they all have `uptime`s of greater than 1 day.  I can try convert a number of the machines to ip based connections to bypass any dns resolution.

**Comment by pshipton** [comment]
Friday night
https://ci.eclipse.org/openj9/job/Test_openjdk11_j9_sanity.system_ppc64le_linux_Nightly/215
ub16p8j95
https://ci.eclipse.org/openj9/job/Test_openjdk11_j9_extended.functional_x86-64_linux_Nightly/200
ub16x64j95
https://ci.eclipse.org/openj9/job/Test_openjdk13_j9_sanity.system_x86-64_linux_Nightly/103
ub16x64j92
https://ci.eclipse.org/openj9/job/Test_openjdknext_j9_extended.functional_ppc64le_linux_OpenJDK/79
ub16p8j93
https://ci.eclipse.org/openj9/job/Test_openjdk11_j9_extended.functional_ppc64le_linux_xl_Nightly/1
ub16p8j96 - timeout running JvmCpuMonitorMXBean
https://ci.eclipse.org/openj9/job/Test_openjdk8_j9_extended.functional_ppc64le_linux_xl_Nightly/1
ub16p8j92 - timeout running JvmCpuMonitorMXBean

Saturday night
https://ci.eclipse.org/openj9/job/Test_openjdk11_j9_sanity.system_x86-64_linux_Nightly/204
ub16x64j95
https://ci.eclipse.org/openj9/job/Test_openjdk11_j9_sanity.system_x86-64_linux_xl_Nightly/207
ub16x64j97
https://ci.eclipse.org/openj9/job/Test_openjdk8_j9_extended.functional_ppc64le_linux_Nightly/215
ub16p8j95
https://ci.eclipse.org/openj9/job/Test_openjdk11_j9_extended.functional_ppc64le_linux_Nightly/217
ub16p8j92
https://ci.eclipse.org/openj9/job/Test_openjdk11_j9_sanity.system_ppc64le_linux_xl_Nightly/2
ub16p8j94
https://ci.eclipse.org/openj9/job/Test_openjdk13_j9_sanity.system_ppc64le_linux_Nightly/113
ub16p8j94
https://ci.eclipse.org/openj9/job/Test_openjdk11_j9_extended.functional_x86-64_linux_Nightly/201
ub16x64j92
https://ci.eclipse.org/openj9/job/Test_openjdk13_j9_extended.functional_ppc64le_linux_Nightly/112
ub16p8j96
https://ci.eclipse.org/openj9/job/Test_openjdk13_j9_sanity.system_x86-64_linux_xl_Nightly/103
ub16x64j92
https://ci.eclipse.org/openj9/job/Test_openjdknext_j9_extended.functional_ppc64le_linux_OpenJDK/80
ub16p8j92
https://ci.eclipse.org/openj9/job/Test_openjdk13_j9_sanity.system_ppc64le_linux_xl_Nightly/2
ub16p8j93
https://ci.eclipse.org/openj9/job/Test_openjdk8_j9_extended.functional_x86-64_linux_xl_Nightly/208
ub16x64j97 - timeout in JvmCpuMonitorMXBean
https://ci.eclipse.org/openj9/job/Test_openjdk8_j9_extended.functional_ppc64le_linux_xl_Nightly/2
ub16p8j95 - timeout in JvmCpuMonitorMXBean

Sunday night
pending

Monday night
pending

**Comment by jdekonin** [comment]
Many of these failures, I stopped confirming after 5, are referencing the ip addresses that the machine use to have.  Each of the machine above had an entry in the /etc/hosts with the old address.  I am not 100% sure if this is required configuration, because on a few of the machines not reported above there was no ip and hostname in the /etc/hosts, so I commented it out on the remaining systems.

Hopefully this cleans up the builds a bit.
